src/account/accountService.ts

```diff
@@ -156,3 +156,41 @@ export const signOut = async () => {
     await Promise.reject(response);
   }
 };
+
+export const checkUserInProject = async (
+  projectId: string,
+  userEmail: string,
+) => {
+  const existQuery = graphql(`
+    query userExists($email: String!) {
+      users(email: $email) {
+        edges {
+          node {
+            ...userFields
+          }
+        }
+      }
+    }
+  `);
+
+  const inProjectQuery = graphql(`
+    query userExistsInProject($project: String!) {
+      users(project: $project) {
+        totalCount
+      }
+    }
```

Comment:
issue: i think this query returns the total number of users in a project, not the whether a given user is in the project?
```suggestion
    query userExistsInProject($email: String!, $project: String!) {
      users(email: $email, project: $project) {
        totalCount
      }
    }
```

Reply:
Haha yes! Thanks